{
	"id": 85168263,
	"body": "If the signal handler gets called somewhere around the call to ```darwin_arm_init_thread_exception_port```, then the signal handler will be run. It will attempt to call ```load_g```, which will fail and that will pass into ```badsignal```.\r\n\r\nIn ```badsignal```, there's a ```cgocallback```. I think that's what's causing this.",
	"user": {
		"login": "crawshaw",
		"id": 161319,
		"type": "User",
		"site_admin": false
	},
	"created_at": "2015-03-23T19:56:39Z",
	"updated_at": "2015-03-23T19:56:39Z"
}
